docs/ABI/Mangling.rst

```diff
@@ -230,6 +229,8 @@ types where the metadata itself has unknown layout.)
   global ::= type generic-signature 'TH' // key path equality
   global ::= type generic-signature 'Th' // key path hasher
   global ::= global generic-signature? 'TJ' AUTODIFF-FUNCTION-KIND INDEX-SUBSET 'p' INDEX-SUBSET 'r' // autodiff function
+  global ::= from-type to-type generic-signature? 'TU' AUTODIFF-FUNCTION-KIND // autodiff self-reordering reabstraction thunk
+  global ::= from-type generic-signature? 'TY' AUTODIFF-FUNCTION-KIND INDEX-SUBSET 'p' INDEX-SUBSET 'r' INDEX-SUBSET 'p' // autodiff subset parameters thunk
```

Comment:
Can you make this a sub-mangling of 'TJ', e.g. 'TJU' and 'TJY'?
We are almost running out of 'T' mangling characters and for auto-diff the symbol length does not really matter.

Reply:
Done. Changed to `TJO` and `TJS`.